lookout/style/format/analyzer.py

```diff
@@ -153,8 +153,11 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_request_st
             trainable_rules = TrainableRules(
                 prune_branches_algorithms=config["prune_branches_algorithms"],
                 prune_attributes=config["prune_attributes"],
-                random_state=config["random_state"],
-                uncertain_attributes=config["uncertain_attributes"], **bscv.best_params_
+                top_down_greedy_budget=config["top_down_greedy_budget"],
```

Comment:
Can you please reorder this change so that there is only 1 removed line here (leave `random_state` and `uncertain` intact)

Reply:
I just this it is better to have the same order we have above: 
https://github.com/src-d/style-analyzer/pull/148/files/9c0c5615965b38a890760c7a2147482c1538cfe8#diff-b71e0e71a29c83c6fa2ee406a76c3baeR125